README.md

```diff
@@ -26,10 +26,9 @@ A workflow visualization tool for [OpenStack Mistral](https://github.com/opensta
 
 ## Requirements
 
-### Mistral >= Pike
-CloudFlow requires Mistral **Pike** or greater, as we rely on
-new [runtime_context](https://docs.openstack.org/developer/mistral/developer/webapi/v2.html#tasks)
-added to Mistral Pike.
+### Mistral >= Stein
```

Comment:
I actually wouldn't change that (as older versions of CloudFlow support older versions of Mistral) but add a table underneath it instructing the users which version to use, like:

| Mistral Version | CloudFlow Version to use |
| -- | -- |
| Stein (or newer) | Latest stable |
| Pike - Rocky | [0.6.4](https://github.com/nokia/CloudFlow/releases/tag/v0.6.4) |

------------
Also, please manually bump the version to `0.7.0` in [package.json](https://github.com/nokia/CloudFlow/blob/master/package.json#L3) and [about.html](https://github.com/nokia/CloudFlow/blob/master/src/app/about/about.component.html#L4) and include those in this PR as well.

(That's the last thing, I swear :))

Reply:
Done